mautic/mautic
# Features+3.3.3

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:

This PR merges changes from version [3.3.3-rc](https://github.com/mautic/mautic/releases/tag/3.3.3-rc) to the features branch.

There are 2 PRs that did not make it as the plugins were moved to a separate repo:
- https://github.com/mautic/mautic/pull/9618
- https://github.com/mautic/mautic/pull/7753

Both targeting https://github.com/mautic/plugin-crm



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. All the PRs from 3.3.3-rc were already tested separately. We just have to make sure that the merge conflicts did not introduce any issues and that the build is passing.
